app/uk/gov/hmrc/rasapi/connectors/FileUploadConnector.scala

```diff
@@ -17,7 +17,7 @@
 package uk.gov.hmrc.rasapi.connectors
 
 import akka.actor.ActorSystem
-import akka.stream.ActorMaterializer
+import akka.stream.{ActorMaterializer, Materializer}
```

Comment:
Unused import but I believe you was likely trying to resolve the deprecated `ActorMaterializer`. However, this implicit val can just be removed now (`matFromSystem` is called instead).

Reply:
Same for `RasfileReader`